src/lib/utils/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {
+  Fade,
+  makeStyles,
+  Tooltip as MuiTooltip,
+  TooltipProps as MuiTooltipProps,
+} from '@material-ui/core'
+
+export type TooltipProps = MuiTooltipProps
+
+const useTooltipStyle = makeStyles(theme => ({
+  arrow: {
+    // gray-700
+    color: '#878E95',
+  },
+  tooltip: {
+    // gray-700
+    backgroundColor: '#878E95',
+    fontSize: '14px',
+    borderRadius: '2px',
+  },
+}))
+
+export default function Tooltip(props: TooltipProps) {
+  const classes = useTooltipStyle()
+  return (
+    <MuiTooltip classes={classes} arrow TransitionComponent={Fade} {...props}>
```

Comment:
Why did you need to wrap the MuiTooltip?  Any idea why the react-tooltip was not behaving properly?
Will this tooltip look different than other tooltips in the application?

Reply:
I wrapped the MUI tooltip to ensure the provided style will always be included, and to give us the flexibility to change the implementation later while still using this component.

The tooltip will look different. I can change it to match the React Tooltip until we can change everything at once.